lib/ledger.js

```diff
@@ -108,17 +109,17 @@ async function _setupGenesisNode() {
 // setup a peer node by fetching the genesis block from another peer
 async function _setupPeerNode() {
   logger.debug('Retrieving genesis block from peers', {peers: cfg.peers});
-  const {strictSSL} = config.jsonld;
-
+  // we need to get the httpsAgent here as it starts out as null.
+  const {httpsAgent} = bedrockHttpsAgent;
+  if (httpsAgent === null) {
+    throw new BedrockError('Bedrock Https Agent is null.', 'InvalidStateError');
```

Comment:
If it's written out, cap HTTPS:
```suggestion
    throw new BedrockError('Bedrock HTTPS Agent is null.', 'InvalidStateError');
```